guide/auto-update.md

```diff
@@ -6,17 +6,18 @@ outline: deep
 # Automatic reload
 
 With this behavior, once the browser detects a new version of your application, then, it will update the caches and will reload any browser windows/tabs with the application opened automatically to take the control.
-
-::: warning
-In order to reload all client tab/window, you will need to import any virtual module provided by the plugin: if you're not using any virtual, there is no way to interact with the application ui, and so, any client tab/window will not be reloaded (the old service worker will be still controlling the application).
-
-Automatic reload is not automatic page reload, you will need to use the following code in your application entry point if you want **automatic page reload**:
```

Comment:
I still do not understand this sentence, I admit. I aggressively dropped this sentence.

Without registering the SW in the application nothing is happening here. That the SW installs any new application caches as soon it is triggered to do so happens independently of the `registerType`.